test/test.js

```diff
@@ -356,7 +358,10 @@ describe('Routes', () => {
       )
       await assertResponseStatus(res, 200)
     })
-    it('ignores duplicate submissions', async () => {
+
+    // Duplicate submissions are ok, because we filter out duplicates in the fraud detection step
+    it('allows duplicate submissions', async () => {
```

Comment:
Do we have any concerns about DoS attacks? Before my change, clients could submit only as many results as many measurements they created. After my change, they can remove as many measurements as they like. I think my change is not making the situation significantly worse than it was before.

Reply:
What do you mean by "clients can remove as many measurements as they like"?

Reply:
Sorry, that was a typo. I meant:
_After my change, they can submit as many measurements as they like._

Reply:
I think it's ok. We can discuss multiple strategies once we are more concerned with production deployment topics, or once we have many clients